src/operators.jl

```diff
@@ -20,16 +20,16 @@ jointerms(terms1::AbstractArray{<:Term}, terms2::AbstractArray{<:Term}) = merges
 
 # Multiplication is handled as a special case so that we can write these
 # definitions without resorting to promotion:
-(*)(x, v::MonomialLike) = Term(x, Monomial(v))
-(*)(v::MonomialLike, x) = Term(x, Monomial(v))
+MP.multconstant(α, v::MonomialLike) = Term(α, Monomial(v))
```

Comment:
I had many trouble with method ambituities on DynamicPolynomials because I was defining `*(constant, t::Term)`, `*(constant, m::Monomial)`, ... You didn't have these issues because you just had `*(constant, t::AbstractPolynomialLike) = *(promote(constant, t)...)` and `*(t::AbstractPolynomialLike, constant) = ...`.
It becomes even more important with RationalPoly since it is not an AbstractPolynomialLike.
I thought that if, for each operation between a constant and a Union{RationalPoly, AbstractPolynomialLike}, I catch the call and then transform it into a function call special for constant op type of our packages, then we won't have any more ambiguities issues.